src/main/java/cat/udl/eps/softarch/geolearning/domain/ImageName.java

```diff
@@ -15,4 +19,9 @@
 
     @OneToMany(mappedBy = "imageName" ,cascade = CascadeType.ALL, orphanRemoval = true)
     private List<ImageNameQuestion> questions;
+
+    @Column
+    @NotNull
+    private Boolean isWrite = false;
```

Comment:
I suppose this new attribute is for something you are implementing in the client side, so I will accept it when you PR the client side too.

Reply:
Okey

Reply:
PR created where we start using boolean isWrite
[Image write](https://github.com/UdL-EPS-SoftArch/GeoLearning/pull/11#issue-422146328)